src/renderer/store/modules/utils.js

```diff
@@ -288,6 +290,14 @@ const actions = {
     //
     // If `urlType` is "invalid_url"
     // Nothing else
+
+    // Invidious doesn't support channel handles yet
```

Comment:
Some instances don't yet but it is now supported as of recently https://yt.artemislena.eu/@linustechtips

Reply:
Revisit once https://github.com/iv-org/invidious/pull/3612 is merged